common/src/main/java/discord4j/common/GitProperties.java

```diff
@@ -15,26 +15,38 @@
  * along with Discord4J. If not, see <http://www.gnu.org/licenses/>.
  */
 
-package discord4j.core.util;
+package discord4j.common;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.Properties;
 
-public final class VersionUtil {
+/**
+ * Provide information about the Git repository version captured at build time.
+ */
+public class GitProperties {
```

Comment:
Also in general I feel the design of this class can be improved. I feel it should be an enum, `System.out.println(GitPropery.APPLICATION_NAME.toString())` would print what you would expect intuitively, while, for myself, I've done this in the past `System.out.println(GitProperty.APPLICATION_NAME)` and it wasn't obvious why. The reliance of `getProperties` to get the correct value I feel is confusing and we can be better. In fact, we could just make its instantiation static, so it's cached, for the enums to work off of.

Reply:
I agree, this was just a package move. We could improve on it later.